randr.c

```diff
@@ -21,15 +21,51 @@
  */
 
 #include "xplugd.h"
+#include "edid.h"
 
 static char *con_actions[] = { "connected", "disconnected", "unknown" };
 
+int get_monitor_name(char *name, char *monitor_name)
+{
+	char path[255];
+	FILE *fp = NULL;
+	size_t result;
+	unsigned char edid[128];
+	MonitorInfo *info;
+
+	snprintf(path, sizeof(path), "/sys/class/drm/card0-%s/edid", name);
+	syslog(LOG_DEBUG, "DRM device sysfs path %s\n", path);
+	fp = fopen(path, "rb");
+	if (!fp) {
+		syslog(LOG_ERR, "Failed to find sys path for %s\n", name);
+		return -1;
+	}
+
+	result = fread (edid, 1, sizeof(edid), fp);
+	if (result != 128) {
+		syslog(LOG_NOTICE, "No EDID data found");
```

Comment:
A user with a faulty display, or similar, would see this message very often.  How about changing this to `LOG_INFO`, or maybe even collapse the two messages into one `LOG_DEBUG`?

When debugging the setup, the first couple of times you run xplugd in a new environment, I'm guessing most people enable all debug they can get.